src/main/java/net/minecraftforge/datafix/ForgeDSL.java

```diff
@@ -0,0 +1,25 @@
+package net.minecraftforge.datafix;
+
+import com.mojang.datafixers.DSL;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.types.Type;
+import com.mojang.datafixers.types.templates.TaggedChoice;
+import com.mojang.datafixers.types.templates.TypeTemplate;
+import net.minecraftforge.datafix.event.DataFixEvent;
+
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class ForgeDSL
```

Comment:
A new class for a single method which could've just as easily been in `ForgeHooks`, since ya know, it's a hook into patched code from Forge?

Reply:
This is ment to expanded in the future and is clearly not a hook but a DFU DSL entry.